server/src/constants.js

```diff
@@ -0,0 +1,3 @@
+const distantFuture = 2147483647;
```

Comment:
ooh shiny this is on client and server. This might end up being our first shared module! (don't do it now but we could maybe move this to `/shared` or something and share some code between the components. I'll experiment at some point.